tools/stores_l10n/add_new_product.md

```diff
@@ -0,0 +1,261 @@
+# How to add a new product to Stores
+
+Before starting, make sure to read the [overview](overview.md) document to get a general idea of what this web app does and its structure.
+
+This document will explain how to start tracking a new project, using Focus for Android as an example.
+
+## Add new product to configuration
+
+Configuration for the Stores web app is in the `Project` class (`app\classes\Stores\Project.php`). The first thing to do it to add the project to `$products_data`, and you’ll need to determine beforehand:
+* How many channels are supported.
+* Code name and displayed name for this product.
+* If the product needs to support App Store (apple) or Play Store (google).
+
+This is the configuration for Focus for Android:
+
+```PHP
+private $products_data = [
+...
+    'focus_android' =>
+        [
+            'channels' => ['release'],
+            'name'     => 'Focus for Android',
+            'store'    => 'google',
+        ],
+...
+];
+```
+
+Then you need to add an entry in `$supported_locales` for each of the channels available. Since Focus for Android only has a release channel, it will be:
+
+```PHP
+private $supported_locales = [
+...
+    'focus_android' => [
+        'release' => [
+            'ar', 'ast', 'az', 'bn-BD', 'ca', 'cs', 'cy', 'de', 'el', 'eo',
+            'es-AR', 'es-CL', 'es-ES', 'es-MX', 'fa', 'fr', 'fy-NL', 'he',
+            'hu', 'hy-AM', 'id', 'it', 'ja', 'kab', 'ko', 'lo', 'nl', 'pl',
+            'pt-BR', 'ru', 'sk', 'sl', 'sq', 'sv-SE', 'th', 'tr', 'uk',
+            'zh-CN', 'zh-TW',
+        ],
+    ],
+...
+];
+```
+
+Note: it the project is tracked on [Webstatus](https://l10n.mozilla-community.org/webstatus/), you can use its [API](https://github.com/mozilla-l10n/webstatus/#available-urls) to get the list of all locales available ([example](https://l10n.mozilla-community.org/webstatus/api/?product=focus-android&txt) for Focus for Android).
+
+The next step is to add which templates and .lang files will be used for this project. In case of Focus for Android there is:
+* `template`: PHP file used to display the full information about this product.
+* `listing`: .lang file containing the strings used in template (description, keywords, etc.).
+* `screenshots`: .lang file used to localize screenshots. It’s distinct from the main .lang file in order to support only a subset of languages.
+
+Since it’s the first launch for Focus for Android, a `whatsnew` entry won’t be needed, but it will be starting from the first update.
```

Comment:
Not much code actually: one variable in template and in the view. Maybe worth a note as a final paragraph before testing.